src/main/java/frc/robot/filters/TurnToggleFilter.java

```diff
@@ -0,0 +1,19 @@
+package frc.robot.filters;
+
+/**
+ *
+ * 
+ * @author aheitkamp
+ */
+public class TurnToggleFilter extends Filter {
+
+    public TurnToggleFilter() {
+    }
+
+    public void filter(DriveInput DI) {
+        if (!enable && !DI.getOverrideYawLock()) {
```

Comment:
double inversion with and can (a) be written more simply
also. parmaeter should be di not DI